lib/apollo/hooks/actions/sendAnswerAndGetNextQuestion.ts

```diff
@@ -0,0 +1,52 @@
+import SendAnswerAndGetNextQuestion from 'graphql/mutations/sendAnswerAndGetNextQuestion.graphql';
+import { useMutation } from '@apollo/client';
+
+import { useNotifier } from 'contexts/NotifierContext';
+import { useCallback } from 'react';
+import Question from 'domain/Question';
+
+type SendAnswerAndGetNextQuestionProps = {
+  gameId: string | number;
+  value: string | undefined;
+};
+
+type SendAnswerAndGetNextQuestionData = {
+  sendAnswerAndGetNextQuestion: {
+    correctAnswerValue: string;
+    question: Question;
+    correctAnswersCount: number;
+  };
+};
+
+type SendAnswerAndGetNextQuestionInputVariable = SendAnswerAndGetNextQuestionProps;
+
+type SendAnswerAndGetNextQuestionVariables = {
+  input: SendAnswerAndGetNextQuestionInputVariable;
+};
+
+const useSendAnswerAndGetNextQuestion = () => {
+  const { setError } = useNotifier();
+
+  const [mutation, sendAnswerAndGetNextQuestionState] = useMutation<
+    SendAnswerAndGetNextQuestionData,
+    SendAnswerAndGetNextQuestionVariables
+  >(SendAnswerAndGetNextQuestion, {
+    onError: (error) => {
+      if (setError) setError(error);
+    },
+  });
+
+  const sendAnswerAndGetNextQuestion = useCallback(
+    ({ gameId, value }: SendAnswerAndGetNextQuestionProps) => {
+      const sendAnswerAndGetNextQuestionInput = { gameId: +gameId, value };
```

Comment:
Why we use type cast? Can we use only one type for gameId?

Reply:
That's true, now we can, we couldn't before. I'll fix it